tests/iteminfo_test.cpp

```diff
@@ -2330,7 +2330,7 @@ TEST_CASE( "repairable and with what tools", "[iteminfo][repair]" )
     // FIXME: Use an item that can only be repaired by test tools
     CHECK( item_info_str( hazmat, repaired ) ==
            "--\n"
-           "<color_c_white>Repair</color> using soldering iron, TEST soldering iron, or extended multitool.\n" );
+           "<color_c_white>Repair</color> using soldering iron, TEST soldering iron, firearm repair kit, gunsmith repair kit, or extended multitool.\n" );
```

Comment:
Just the order I think.
```suggestion
           "<color_c_white>Repair</color> using gunsmith repair kit, firearm repair kit, soldering iron, TEST soldering iron, or extended multitool.\n" );
```

Reply:
Thanks for fixing it!